proto/scheduler.proto

```diff
@@ -84,7 +84,12 @@ message SchedulingMetadata {
   string os = 2;
   string arch = 3;
   string pool = 4;
-  string group_id = 5;
+  // Group ID that owns the executors on which the task is to be executed.
+  // May be different from the Group ID of the user that issued the Execute
+  // request.
+  string executor_group_id = 5;
+  // Group ID of the user that issued the Execute request.
+  string task_group_id = 6;
```

Comment:
since these are group_ids, not api keys -- are these only different for our executors, or will they ever be different in other cases?

Reply:
They should only be different for our executors.